tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev

```diff
@@ -0,0 +1,11 @@
+ARG AGENT_BASE=datadog/agent:latest
+FROM $AGENT_BASE
+
+# include some useful dev tools since this will be used or development
+RUN apt-get update -y && apt-get install -y jq conntrack netcat dnsutils
+
+# inv -e process-agent.build-dev-image will set up a temporary
+# build directory where this Dockerfile, and the necessary binaries
+# are in the same directory
+COPY process-agent /opt/datadog-agent/embedded/bin/process-agent
+COPY system-probe /opt/datadog-agent/embedded/bin/system-probe
```

Comment:
maybe you could also overwrite the default  `CMD` to execute by default `process-agent`

Reply:
Usually installing a dev build of the agent, we want the whole agent running.   That way host metadata/metrics get reported.